src/components/charts/chart/chart.md

```diff
@@ -100,6 +100,62 @@ const toggleChartType = (filtersSelected) => {
 </React.Fragment>
 ```
 
+
+Example with projected data
+```js
+const data = require('../assets/data.js');
+const projectedData = require('../assets/projected-data.js');
+initialState = {
+  ...data,
+  config: { ...data.config, ...projectedData.config },
+  projectedData: projectedData.data,
+  type: 'line',
+  loading: false
+};
+const chartTypes = ['line', 'area', 'percentage'];
+const handleLegendChange = (filtersSelected) => {
+  setState(state => {
+    const filterIds = filtersSelected.map(f => f.label);
+    const filteredColumns = data.initialYColumns.filter(col => filterIds.includes(col.label));
+    const config = {...state.config};
+    config.columns.y = filteredColumns;
+    return {
+      filtersSelected,
+      config
+    }
+  })
+}
+const toggleChartType = (filtersSelected) => {
+  setState(state => {
+    const chartTypeIndex = chartTypes.indexOf(state.type);
+    let type = chartTypes[0];
+    if (chartTypeIndex + 1 < chartTypes.length) {
+      type = chartTypes[chartTypeIndex + 1]
+    }
+    return { type };
+  });
+}
+<React.Fragment>
+  <Button onClick={toggleChartType} >
+    Toggle type
+  </Button>
+  <Chart
+    type={state.type}
+    config={state.config}
+    data={state.data}
+    projectedData={state.projectedData}
```

Comment:
I know this `prop` is passed on `Chart` but consumed by a child component (`Line` or `Area` charts), but would be great have it documented in the parent component as is the one that we are going to be using on each project.
Knowing the `shape` of the `data` and `config` of `projectedData` without needing to dive into the code would be great.
In the example this are the PropTypes:
```js
projectedData: PropTypes.shape({
  data: PropTypes.arrayOf(PropTypes.shape({
    x: PropTypes.number,
    y: .oneOfType([ PropTypes.number, PropTypes.arrayOf(PropTypes.number) ])
    label: PropTypes.string,
    isRange:PropTypes.bool
  })),
  config: PropTypes.shape({
    dividerLine: PropTypes.shape({
      pre: PropTypes.string,
      post: PropTypes.string
    })
  })
})
```